src/components/network/network-map.js

```diff
@@ -253,7 +253,9 @@ const NetworkMap = (props) => {
     function renderTooltip() {
         return (
             tooltip &&
-            tooltip.visible && (
+            tooltip.visible &&
```

Comment:
```suggestion
            tooltip?.visible &&
```

then remove tooltip && (line 255)

Reply:
Done